src/js/subtraction/components/__tests__/SubtractionList.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createBrowserHistory } from "history";
+import React from "react";
+import { describe, expect, it } from "vitest";
+import { createFakeAccount, mockGetAccountAPI } from "../../../../tests/fake/account";
+import { createFakeSubtractionMinimal, mockApiGetSubtractions } from "../../../../tests/fake/subtractions";
+import { renderWithRouter } from "../../../../tests/setupTests";
+import { AdministratorRoles } from "../../../administration/types";
+import SubtractionList from "../SubtractionList";
+
+describe("<SubtractionList />", () => {
+    let history;
+    let subtractions;
+
+    beforeEach(() => {
+        subtractions = createFakeSubtractionMinimal();
+        history = createBrowserHistory();
+    });
+
+    it("renders correctly", async () => {
+        const scope = mockApiGetSubtractions([subtractions]);
+        renderWithRouter(<SubtractionList />, {}, history);
+
+        await waitFor(() => expect(screen.queryByLabelText("loading")).not.toBeInTheDocument());
+        expect(screen.getByText("Subtractions")).toBeInTheDocument();
+        expect(screen.getByText("1")).toBeInTheDocument();
+        expect(screen.getByText(subtractions.name)).toBeInTheDocument();
+
+        scope.isDone();
```

Comment:
```suggestion
        scope.done();
```
That is if your trying to check for all API routes having been called.